python-scripts/xbox/interface/if_xbdm.py

```diff
@@ -190,11 +199,12 @@ def write1(address, data, physical):
 
   from xbox.pe import *
 
-  modules = GetModules()
-  # FIXME: Get location of xbdm.dll
-  xbdm_base = 0x0B0011000
-  DmResumeThread_addr = resolve_export(35, image_base=xbdm_base) #FIXME: Use location of xbdm.dll
-  #DmResumeThread_addr = resolve_export(35) #FIXME: Use location of xbdm.dll
+  modules=GetModules() #init modules
+
+  xbdm_base = modules["xbdm.dll"]["base"]
```

Comment:
With suggestion from above this would get something like:

```python
modules = [{'name': 'xbdm.dll', 'base': 3735928559, 'tls': True}, {'name': 'foo.exe', 'base': 9001}]

xbdm = [module for module in modules if module['name'] == "xbdm.dll"][0]
print(xbdm['base'])
```